docs/src/pages/Conformance.md

```diff
@@ -248,6 +248,13 @@ Reference searches on the IBM FHIR Server support search on:
 
 We recommend using logical reference where possible.
 
+Elements of type Reference may contain a versioned reference, such as `Patient/123/_history/2`. When performing chained, reverse chained (`_has`), `_include`, or `_revinclude` searches on versioned references, the following rules apply:
+
+* If a resource has a reference that is versioned, and a chained search is performed using the element containing the versioned reference, the search criteria will be evaluated against the current version of the referenced resource, regardless of the version specified. For example, if a `Condition` resource contains a reference of `Patient/123/_history/1` in its `subject` element, and the current version of the `Patient/123` resource is 2, and a search of `Condition?subject.name=Jane` is performed, the search criteria will be evaluated against the current version of the `Patient/123` resource rather than the specified version. This is because the IBM FHIR Server only stores search index information for the current versions of resources. In the case where a chained search does not act on the referenced version of a resource, the search results will contain an `OperationOutcome` with a warning that indicates the logical id of the resource and the element containing the versioned reference.
```

Comment:
A small suggestion as this rule is very dense.

Establish the Rule
Constraints/Outcomes of the Rule
Example

It's so very dense.

Reply:
I did this for each search type. Hopefully a little more readable/digestable.